### ISSUE TITLE
Minimap no longer shows current position in the file

### ISSUE THREAD

**Issue by DFelten** [microsoft/vscode#165792]
Type: <b>Bug</b>

The scroll indicator on the minimap is missing. The minimap no longer shows the current position in the file. You can still scroll with it, but you can't see where you are in the file.

<img width="149" alt="CleanShot 2022-11-08 at 09 07 04@2x" src="https://user-images.githubusercontent.com/1169185/200509015-a4e0acc1-489d-4655-bd92-ea62f64cac0e.png">

VS Code version: Code - Insiders 1.74.0-insider (2282be522a8031209a739783baede67bfa9fc95a, 2022-11-08T05:33:58.303Z)
OS version: Darwin arm64 22.1.0
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|4, 4, 5|
|Memory (System)|32.00GB (0.27GB free)|
|Process Argv|--crash-reporter-id 511166e9-0c5e-4225-a6af-efd2d90c3f8d|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (79)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.2
codesnap|adp|1.3.4
vscode-m3u|af4|1.0.0
Bookmarks|ale|13.3.1
project-manager|ale|12.7.0
flutter-snippets|ale|3.0.0
toggle-excluded-files|eam|1.7.0
clipboard-history|Anj|1.0.7
melos-code|bla|0.6.0
vscode-intelephense-client|bme|1.8.2
ccs-flutter-color|cir|1.0.2
gitignore|cod|0.9.0
php-namespace-resolver|ctf|0.0.9
vscode-mac-color-picker|dae|1.1.0
dart-code|Dar|3.53.20221107
flutter|Dar|3.52.0
git-diff-and-merge-tool|dav|1.0.5
vscode-markdownlint|Dav|0.48.1
vscode-new-file|dku|4.0.2
githistory|don|0.6.19
gitlens|eam|13.0.4
bloc|Fel|6.6.2
mason|Fel|0.1.6
flutter-coverage|flu|0.0.4
vscode-pull-request-github|Git|0.54.0
todo-tree|Gru|0.0.220
terraform|has|2.24.3
json-to-dart|hir|3.5.8
rest-client|hum|0.25.1
monkey-patch|ioc|0.1.23
pubspec-assist|jer|2.3.2
shortcut-menu-bar|jer|3.0.4
intellij-idea-keybindings|k--|1.5.4
indent-to-bracket|LAK|1.2.2
flutter-intl|loc|1.18.2
syntax-project-pbxproj|mar|0.1.3
flutter-snippets|Mar|1.0.5
Kotlin|mat|1.7.1
rainbow-csv|mec|3.3.0
git-graph|mhu|1.30.0
dart-barrel-file-generator|miq|4.0.0
diff-merge|mos|0.7.0
vscode-docker|ms-|1.22.2
vscode-language-pack-de|MS-|1.73.11020948
isort|ms-|2022.4.0
python|ms-|2022.18.1
vscode-pylance|ms-|2022.11.10
jupyter|ms-|2022.11.1003131031
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.12
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.263.0
live-server|ms-|0.4.3
vsliveshare-pack|ms-|0.4.0
vscode-flutter-freezed-helper|mth|0.1.0
sqltools|mtx|0.25.1
sqltools-driver-mysql|mtx|0.4.0
awesome-flutter-snippets|Nas|4.0.1
vscode-yaml-sort|Pas|6.0.0
vscode-versionlens|pfl|1.0.10
material-icon-theme|PKi|4.22.0
remove-comments|pli|1.2.2
vscode-thunder-client|ran|1.20.1
vscode-yaml|red|1.10.1
vscode-sort-json|ric|1.20.0
vscode-coverage-gutters|rya|2.10.2
lcov-lang|Ser|1.0.1
reveal|smu|1.2.7
swift-lang|ssw|0.9.0
freezed-model-gen|Sus|1.0.1
better-search|tra|0.0.6
errorlens|use|3.6.0
vscode-lldb|vad|1.8.1
vscode-todo-highlight|way|1.0.5
php-debug|xde|1.29.0
vscode-surround|yat|1.5.0
markdown-all-in-one|yzh|3.4.3
material-theme|zhu|3.15.6

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30422396
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805:30301674
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30404738
cppdebug:30492333
vsclangdf:30492506
c4g48928:30535728
dsvsc012cf:30540253
pylantcb52:30590116
vscccc:30564267
pyindex848cf:30577859
nodejswelcome1:30587009
gswce1:30602633
34cci188:30599831
pyind779cf:30600160
dbltrim-noruby:30604474

```

</details>

**Comment by AshSimmonds** [comment]
Yeah that's painful. Not exactly a workaround, but something to make things easier until there's a fix is to make the scrollbar super fat.  
  
`"editor.scrollbar.verticalScrollbarSize": 24`

**Comment by AshSimmonds** [comment]
This could be related? Added all the options to `settings.json` and some of the hex colour previews aren't rendering:  
  
![ON24Sm7zog](https://user-images.githubusercontent.com/9034041/200656735-39de97e9-87e8-4c70-bbfd-8567130f6d83.png)

**Comment by mikiqex** [comment]
Another Not exactly a workaround:
- Help / Toggle Developer Tools
- On Elements find the Minimap Slider (you can search for class "`minimap-slider-horizontal`")
- In Styles for `.monaco-editor .minimap-slider .minimap-slider-horizontal` change "color" to "background" (click on "color" and rewrite)

Final result:
```css
.monaco-editor .minimap-slider .minimap-slider-horizontal {
    background: var(--vscode-minimapSlider-background);
}
```

I've just updated to commit 96db0b and it's not yet fixed.

**Comment by cip8** [comment]
> Final result:
> 
> ```css
> .monaco-editor .minimap-slider .minimap-slider-horizontal {
>     background: var(--vscode-minimapSlider-background);
> }
> ```
> I've just updated to commit 96db0b and it's not yet fixed.

Thanks @mikiqex - this works great as a temporary workaround.

**Comment by GavinRay97** [comment]
For what it's worth, I think this is the first noticeable bug/regression I've had in several years of using Insiders. I'll take it.

**Comment by sanket-bhalerao** [comment]
It feels way weirder than I imagined. i disabled the scrollbar in favor of the minimap and slider, now I scroll like a lost person 🤣

**Comment by cip8** [comment]
> It feels way weirder than I imagined. i disabled the scrollbar in favor of the minimap and slider, now I scroll like a lost person 🤣

😂 I also disabled the scrollbar altogether & I'm at a loss now. 

My temporary fix was overwritten by a fresh update & yet the minimap is still broken 😔
I'll switch to the vanilla version for now as constantly applying manual patches is too time consuming.

I'm a bit surprised the @VSCode team ignores this easy _(but very important)_ fix & that no one in their team noticed a huge feature suddenly gone 🤷‍♂️

**Comment by Eldair** [comment]
Yeah, them not acknowledging or commenting on this is very strange 🤨

**Comment by ghylander** [comment]
There's been a lot of issues it seems, I didn't know this UI element was called minimap and couldn't find these issues before.
Latest update (commit #d9a70c4) didn't fix it.
@mikiqex's solution worked and restored the exact behaviour.
I'm curious as to why it hasn't been fixed yet.

**Comment by AshSimmonds** [comment]
This is frustrating as being "bleeding edge" I get updates almost daily, I understand the position I'm in, however this one which overrides my personal workflow significantly is just disappointing to have to fix frequently.  
  
- https://youtu.be/hc4je-T8cDs

**Comment by ghylander** [comment]
Ok, just noticed something else that is no longer shown
Took me a while to notice. I was feeling something else was off but couldn't quite put my finger on it.

I enabled editor rules, which are shown as vertical white-ish lines marking user-specified widths.
They are gone as well

**Comment by cip8** [comment]
> Ok, just noticed something else that is no longer shown Took me a while to notice. I was feeling something else was off but couldn't quite put my finger on it.
> 
> I have had editor rules enabled for a time now, which are shown as vertical white-ish lines marking user-specified widths. They are gone as well.

Maybe Elon bought them too? 🤔

**Comment by sanket-bhalerao** [comment]
let's not be negative here. 
I get that those who explicitly use and love `minimap` are upset due to not being able to use it properly, however, let's cut the VSCode team some slack. As you would agree they are doing exceptional work and bringing us awesome features with every release.
Few of the participants in the conversation have already found a fix for the issue, maybe you could raise a PR and get your contributor badge😋 
let's be patient, something awesome might be coming our way soon 🥳

**Comment by sanket-bhalerao** [comment]
let's not be negative here. 
I get that those who explicitly use and love `minimap` are upset due to not being able to use it properly, however, let's cut the VSCode team some slack. As you would agree they are doing exceptional work and bringing us awesome features with every release.
Few of the participants in the conversation have already found a fix for the issue maybe you could raise a PR and get your contributor tag ready 😋 
lets be patient, something awesome might be coming our way soon 🥳

**Comment by alexdima** [comment]
Fixed via https://github.com/microsoft/vscode/pull/166691

**Comment by jerryc05** [comment]
FYI I'm currently using
```
Version: 1.74.0-insider (user setup)
Commit: 1b1e2a2c5794df1f475976062c754651827634a9
Date: 2022-11-18T05:25:18.342Z
Electron: 19.1.3
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes
```
and the slider still seems not showing correctly

**Comment by zedd3v** [comment]
@alexdima 

Version: 1.74.0-insider
Commit: 1b1e2a2c5794df1f475976062c754651827634a9
Date: 2022-11-18T05:24:51.315Z
Electron: 19.1.3
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.1.0
Sandboxed: Yes

**Comment by ghylander** [comment]
seems to be fixed for me as of commit #fef85ea792f6627c83024d1df726ca729d8c9cb3

**Comment by jerryc05** [comment]
Confirmed fixed as of 
```
Version: 1.74.0-insider (user setup)
Commit: f730160c5d5fe36a7ad8db51f74f40a9d316e8b2
Date: 2022-11-22T05:21:41.060Z
Electron: 19.1.3
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes
```
